src/pages/content/word-list.mdx

```diff
@@ -1,12 +1,707 @@
+transitive verb ("CC me"), or as an adjective ("the CCed agent").
+
+In documentation, it’s permitted to use lowercase and apostrophes (cc, cc’s,
+cc’ed).
+
+### channel
+
+Use lowercase for general references. Capitalize when referring to t`he Channel
+framework or describing features of the framework, such as the Channel pull
+endpoint.
+
+### chat, Zendesk Chat
+
+Lowercase when talking about a chat message or the verb. Capitalize when
+referring to the feature or the product, such as Zendesk Chat.
+
+### choose, select
+
+Use choose when the user must make a decision. Use select when the user is
+presented with an item from a list to carry out a decision already made.
+
+### click, swipe, tap
+
+Avoid using platform-specific words like click, swipe, or tap.
+
+Use click to describe the action of pressing a key, mouse, or button. For mobile
+or screen-based physical contact, use tap. Pair click directly with the button
+or menu label without “on” or “at.”
+
+<BestPractice>
+  <Do>
+    <p>Preview document</p>
+  </Do>
+  <Dont>
+    <p>Click on preview</p>
+  </Dont>
+</BestPractice>
+
+### Customer Support
+
+Use Zendesk Customer Support or Customer Support when referring to the place
+Zendesk customers go for support on our product. Avoid Support team.
+
+### CSAT
+
+All caps. Short for customer satisfaction. Use only as an adjective, as in CSAT
+rating or CSAT survey.
+
+### data
+
+Always use data with a singular verb. In other words, “the data is...”, “the
+data goes...” whether you mean a collection of facts (plural) or information
+(singular). If you want to emphasize that something is plural, rewrite to use a
+term such as facts or numbers.
+
+### data center
+
+Data center is always two words.
+
+### deselect, cancel
+
+Use clear or cancel instead of deselect. As in, “Clear these checkboxes” and
+“Cancel this selection.”
+
+### done
+
+Avoid done since it doesn’t provide enough information. Button labels should
+make it clear what’s going to happen when the user clicks it.
+
+<BestPractice>
+  <Do>
+    <p>
+      Description: Settings updated <br />
+      Button label: Go to Admin Center
+    </p>
+  </Do>
+  <Dont>
+    <p>
+      Description: Settings updated
+      <br />
+      Button label: Done
+    </p>
+  </Dont>
+</BestPractice>
+
+### drop-down
+
+This noun describes a UI element that lets users pick from a series of options.
+Always hyphenate it.
+
+### for example, e.g.
+
+Keep it conversational by using for example. The scholarly way to say “for
+example” is by using “e.g.,” which stands for exempli gratia.
+
+### either
+
+Use either only for a decision between two choices.
+
+<BestPractice>
+  <Do>
+    <p>You may choose either bacon or tofu.</p>
+  </Do>
+  <Dont>
+    <p>You may choose either bacon, snake meat, or tofu.</p>
+  </Dont>
+</BestPractice>
+
+### email, emails
+
+No hyphen needed.
+
+### feedback
+
+Use give feedback or for some Early Access Program (EAP) links, share your
```

Comment:
Use _give feedback_ or for some Early Access Program (EAP) links, _share your
thoughts_.